laravel/resources/views/modals/fonts/modal-add.blade.php

```diff
@@ -0,0 +1,50 @@
+<div class="modal fade" id="addFont" tabindex="-1" aria-labelledby="addFontLabel" aria-hidden="true">
+    <div class="modal-dialog modal-xl">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h1 class="modal-title fs-5 fw-bold" id="addFontLabel">Add new font</h1>
+                <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+            </div>
+            <form method="POST" action="{{ route('administration.font.create') }}" enctype="multipart/form-data" class="row g-3 needs-validation" novalidate>
+                @csrf
+                <div class="modal-body">
+                    <div class="row">
+                        <div class="col-lg-12">
+                            <div class="alert alert-primary" role="alert">
+                                <p>This project uses TrueType Fonts (TTF) for writing the text on the banner images.</p>
```

Comment:
[CLARIFICATION]

Should we also deduplicate this code, which explains here how you can install fonts?

The same text exists for adding and editing fonts.

Reply:
In my tests i ask me: why should i edit a `font.tt`
What can happen? In the case you have a `small.ttf` and a `big.ttf` and you change form small to big then the drawed font 
will not match with your settings because the letter width / height are different. 

You can check this case. In my opinion we should not use a edit font style.

Reply:
Even when it doesn't happen that often, it can be that there is a newer version of a font due to minimal changes (e. g. new characters, adjusted / fixed existing characters like incorrect style, etc.).

Deleting a font results in deleting all associated configurations (texts), which means that you would need to reconfigure all of these texts after uploading the new version of the font again.

Thus, you're able to update an existing font in order to simply replace it.